purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -880,7 +880,12 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
             val backend = Backend(
                 apiKey,
                 Dispatcher(service),
-                HTTPClient(languageTag = context.getLocale().toBCP47())
+                HTTPClient(
+                    appConfig = AppConfig(
+                        context.getLocale().toBCP47(),
+                        context.packageManager.getPackageInfo(context.packageName, 0).versionName
```

Comment:
Any chance this can be null? Or locale can be null?

Just don't want any NPEs. I guess not cause it's kotlin?

Reply:
Good point, Android code nullability annotations are questionable. I am going to wrap it just in case